### DISCUSSION TITLE
CORS ERROR

### DISCUSSION THREAD

**Discussion by YevheniiVolosiuk** [serversideup/docker-php#503] [Q&A] [answered]
I am using a 

`FROM serversideup/php:7.4-fpm-apache AS base`

and on my wordpress I get an error CORS error
![image](https://github.com/user-attachments/assets/8735304d-0197-486e-99c7-757c74274fc7)

How can I resolve it?

I tried to fix it with security.conf file in Apache but doesn't help

```
#
# List only trusted origins.
#
Header always set Access-Control-Allow-Origin "https://there-be-api-url"
```

Do you know how to fix it or which way to go with it?
@jaydrogers

**Comment by jaydrogers** [comment]
CORS always confuses me, so I usually need to loop in @danpastori because there are usually things you need to do on the app side of things to get it to work.

If anyone else in the community has ideas, it would be greatly appreciated if you chimed in 🙌

**Comment by danpastori** [comment]
CORS is a mega pain. If I were to take a guess is there's something in your PHP/WP install that's not adding the proper Access-Control-Allow-Origin header to be set correctly. I've only managed CORS through code(at least with Laravel).  If it is server-side though, I'd have to throw this back into the discussion.

**Comment by YevheniiVolosiuk** [comment, marked as answer]
This was a problem on an endpoint side API URL. They have a white list for the domain.

Regular staff in WordPress must work ok.

**Reply by jaydrogers** [reply]
Thanks for the update!

This is why whenever I see that error I ping @danpastori. It's usually him and I swearing and the screen together for 45 minutes until there is a solution 🤣

Most the time in my experience, it comes from something weird like that.
